examples/levelset/levelset_periodic.py

```diff
@@ -0,0 +1,68 @@
+from firedrake import *
+from fireshape import *
+import numpy as np
+import ROL
+
+d = 3
+
+# goal: update the mesh coordinates of a periodic mesh
+# so that a function defined on the mesh becomes a given one
+if d == 2:
+    mesh = PeriodicUnitSquareMesh(30, 30)
+elif d== 3:
+    mesh = PeriodicUnitCubeMesh(20, 20, 20)
+Q = PeriodicControlSpace(mesh)  #how can we fix the boundary?
```

Comment:
Is this a question that should be answered before merging this?

Reply:
@APaganini Is this addressed by the new ElastictyInnerProduct? Or marking the plex?

Reply:
This comment has more to do with PETSc and Firedrake then Fireshape. I'll remove it now.